src/api/app/models/bs_request_action.rb

```diff
@@ -724,6 +724,10 @@ def expand_target_project
     self.target_project = tpkg.project.name
   end
 
+  def modify_sources(force_branching)
+    # only used in incidents so far
```

Comment:
please raise `AbstractMethodCalled` if it is an abstract method (which should implement in the subclasses